scripts/release-notes.js

```diff
@@ -1,12 +1,13 @@
 const { execSync } = require('child_process');
 const http = require('https');
 const readline = require('readline');
+const chalk = require('chalk');
 
 const bcd = require('..');
 
 const { argv } = require('yargs')
   .command('$0 <version-tag>', 'Initiate a release of this package on GitHub', (yargs) => {
-    yargs.positional('version-tag', {
+    return yargs.positional('version-tag', {
```

Comment:
This doesn't seem like a `chalk` change either (don't know why I didn't notice it before, whoops).  What's the purpose of this, and can we move this into a separate PR?

Reply:
This is actually a bug in DefinitelyTyped: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35234.